### ISSUE TITLE
About the DIV2K test dataset

### ISSUE THREAD

**Issue by HolmesShuan** [thstkdgus35/EDSR-PyTorch#77]
Hi, the performance of EDSR is impressive, and I am new to super-resolution.
Here, I have two simple questions:
1.  README declares that:

> We measured PSNR using DIV2K 0801 ~ 0900 ...

But the test dataset in DIV2K, which I download from the official link, contains only **0901~1000**. Is that a problem to reproduce the reported results in README? Or the performance of super-resolution methods is consistent? 

2.  The DIV2K pre-trained model can be directly used on Set5, Set14, etc. or fine-tuning is required? Is this a domain adaptation problem?

Thanks~

**Comment by HolmesShuan** [comment]
Please forgive me for my carelessness. After a quick view of your codes, I find `--data_range` argument.
I leave this problem open for the authors to confirm and close.

**Comment by Yannis1995** [comment]
Where did you download the DIV2K constaining 0901~1000?

**Comment by HolmesShuan** [comment]
Please refer to the [NTIRE 2017 competition](https://competitions.codalab.org/competitions/16303#participate).

**Comment by huaixushiliu** [comment]
> Where did you download the DIV2K constaining 0901~1000?

Hello，have you resolved this question？I didn’t find the  DIV2K constaining 0901~1000 too.

**Comment by HolmesShuan** [comment]
FYI.
The [official download link](https://cv.snu.ac.kr/research/EDSR/DIV2K.tar) contains images from 1st to 900th. The training dataset consists of 800 images (0001~0800). 0801~0900 images are used as a validation set. 0901~1000 are not included in the above download link, but can still be downloaded from [NTIRE 2017 competition](https://competitions.codalab.org/competitions/16303#participate). However, since 0901~1000 is the test set, the ground truths (i.e., high-resolution images) have not been released.
